.pre-commit-config.yaml

```diff
@@ -36,3 +36,12 @@ repos:
       exclude: (\.min\.js$|\.svg$|\.html$)
       additional_dependencies:
       - tomli
+-   repo: local
+    hooks:
```

Comment:
Will this exit non-zero if changes were made by the script?

Reply:
pre-commit seems to be smart enough to prevent committing when executing the script leads to changes in the repo.